docs/guide/howto.md

```diff
@@ -0,0 +1,234 @@
+## Stratis how-to/walk-through ##
+
+
+### Introduction ###
+
+[Stratis](https://github.com/stratis-storage)
+(which includes stratisd as well as stratis-cli), provides ZFS/Btrfs-style
+features by integrating layers of existing technology: Linux's devicemapper
+subsystem, and the XFS filesystem.  The [stratisd](https://github.com/stratis-storage/stratisd)
+daemon manages collections of block devices, and provides a
+[D-Bus API](https://stratis-storage.github.io/DBusAPIReference.pdf)
+(not finalized).  The [stratis-cli](https://github.com/stratis-storage/stratis-cli)
+provides a command-line tool `stratis` which itself uses the D-Bus API to
+communicate with stratisd.
+
+Basic workflow is take one or more empty disks and create a Stratis pool.  Using
```

Comment:
Please omit this paragraph. It reads awkwardly and doesn't help explain the steps you show next, installation, etc.